File: DSA/PriyankaSingh_590016827/day16question2.c
#include <stdio.h>

int main() 
{
    int n, target, count = 0;

    // Input array size
    printf("Enter the number of elements in the array: ");
    scanf("%d", &n);

    int arr[n];

    // Input array elements
    printf("Enter %d elements:\n", n);
    for (int i = 0; i < n; i++) 
    {
        scanf("%d", &arr[i]);
    }

    // Input the target value to count
    printf("Enter the target value to count: ");
    scanf("%d", &target);

    // Counting the frequency of the target value
    for (int i = 0; i < n; i++) 
    {
        if (arr[i] == target) 
        {
            count++;
        }
    }

    printf("The target value %d appears %d time(s) in the array.\n", target, count);

    return 0;
}

File: DSA/Prabhjeet_590013286/Day16/Day_16_Question_2.py
def fn(arr, target):
    count = 0
    for num in arr:
        if num == target:
            count += 1
    return count


arr_input = input("Enter the array elements separated by spaces: ")
arr = list(map(int, arr_input.strip().split()))
target = int(input("Enter the target number to count: "))
result = fn(arr, target)
print(f"The target number {target} appears {result} time(s) in the array.")
